<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<<<<<<< HEAD
<testsuite hostname="Pitiass-MBP.fios-router.home" ignored="0" name="Default test" tests="1" failures="0" timestamp="2020-09-18T19:02:49 EDT" time="10.219" errors="0">
  <testcase name="upload" time="3.282" classname="gmail.pitias4work.com.Upload"/>
=======
<testsuite hostname="Pitiass-MBP.fios-router.home" ignored="0" name="Default test" tests="18" failures="1" timestamp="2020-09-17T20:50:56 EDT" time="262.453" errors="0">
  <testcase name="JavaScriptError" time="0.101" classname="take.homw.script.Script"/>
  <testcase name="checkbox" time="0.243" classname="take.homw.script.Script"/>
  <testcase name="context_menu" time="9.429" classname="take.homw.script.Script"/>
  <testcase name="dragandDrop" time="5.549" classname="take.homw.script.Script"/>
  <testcase name="dropDown" time="3.289" classname="take.homw.script.Script"/>
  <testcase name="dynamic_Content" time="18.943" classname="take.homw.script.Script"/>
  <testcase name="dynamic_Control" time="29.579" classname="take.homw.script.Script"/>
  <testcase name="dynamic_Loading" time="8.304" classname="take.homw.script.Script"/>
  <testcase name="fileDownload" time="3.242" classname="take.homw.script.Script"/>
  <testcase name="fileUpload" time="5.407" classname="take.homw.script.Script"/>
  <testcase name="floating_Menu" time="3.211" classname="take.homw.script.Script"/>
  <testcase name="iframe" time="3.503" classname="take.homw.script.Script"/>
  <testcase name="javaScriptAlerts" time="12.456" classname="take.homw.script.Script"/>
  <testcase name="login_Success" time="2.502" classname="take.homw.script.Script"/>
  <testcase name="login_fail" time="2.436" classname="take.homw.script.Script">
    <failure type="java.lang.AssertionError" message="expected [http://localhost:7080/secure] but found [http://localhost:7080/login]">
      <![CDATA[java.lang.AssertionError: expected [http://localhost:7080/secure] but found [http://localhost:7080/login]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at gmail.pitias4work.com.Login_failer.login_fail(Login_failer.java:35)
at take.homw.script.Script.login_fail(Script.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- login_fail -->
  <testcase name="mouse_Hover" time="9.629" classname="take.homw.script.Script"/>
  <testcase name="notification_Msg" time="11.619" classname="take.homw.script.Script"/>
  <testcase name="openNewTab" time="3.298" classname="take.homw.script.Script"/>
>>>>>>> branch 'master' of https://github.com/PitiasFessahaie/take-home-webdriver-test.git
</testsuite> <!-- Default test -->
